bithubph/bithub-fe
# 172099936 setup test with react testing library

#### What does this PR do?
- This PR sets up a test using the react testing library for the application

#### Description of tasks to be completed?
- install `react-testing-library` and `jest-dom`
- add a test script for `index.js`

#### How should this be manually tested?
  - Clone the repo with `clone https://github.com/bithubph/bithub-fe.git`
  - Checkout to branch `ch-setup-react-testing-library-172099936`
  - Run application `npm test`
  - The test result will appear on the terminal:
          `Test Suites: 1 passed, 1 total`
          `Tests:  1 passed, 1 total`

#### Any background context you want to provide?
- You would need npm or yarn to run this application

#### What are the relevant pivotal tracker stories?
- PT Story link - [#172099936](https://www.pivotaltracker.com/n/projects/2440925/stories/172099936)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/19293902/78192345-d9a6ca80-746f-11ea-9679-88e467ef6e80.png)
